av1-rtp-spec.md

```diff
@@ -292,9 +292,11 @@ The following packetization grouping would not be allowed, since it combines dat
 
 If a packet contains an OBU with an OBU extension header then the entire packet is considered associated with the layer identified by the temporal_id and spatial_id combination that are indicated in the extension header. If a packet does not contain any OBU with an OBU extension header, then it is considered to be associated with all operating points.
 
-A MANE or SFU performs its function based on a target operating point. A packet MUST be forwarded if it is associated with the target operating point, or with all operating points. A packet SHOULD NOT be forwarded if it is associated with an operating point different than the target operating point.
+The general function of a MANE or SFU is to selectively forward packets to receivers. To make forwarding decisions a MANE inspects the media payload, so that it needs to be able to parse the AV1 bitstream and cannot function when end-to-end encryption is enabled. An SFU does not parse the AV1 bitstream and therefore needs to obtain the information relevant to selective forwarding by other means, such as the Dependency Descriptor described in Appendix A. In addition to enabling bitstream-independent forwarding and support for end-to-end encryption, the Dependency Descriptor also enables forwarding where the metadata OBU provided in the AV1 bitstream is not sufficient to express the structure of the stream.
 
-SFUs can operate using end-to-end encryption, i.e., with encrypted payload, using the RTP header extension defined in Appendix A. The extension exposes the layer information of a packet so that the SFU can make the appropriate forwarding decision.
+While SFU implementations that support both the Dependency Descriptor and the framemarking RTP header extension [framemarking] MAY advertise support for both header extensions in an Offer, an implementation of this specification that supports the Dependency Descriptor SHOULD only indicate support for the Dependency Descriptor in an Answer. In situations where support for both the Dependency Descriptor and the framemarking RTP header extension are indicated in both the Offer and Answer, implementations of this specification MUST only send packets including the Dependency Descriptor. 
+
+Implementations of this specification that support the framemarking RTP header extension but lack support for the Dependency Descriptor SHOULD indicate support for the framemarking RTP header extension in an Offer or Answer. When the AV1 RTP payload format is negotiated along with the framemarking RTP header extension, the framemarking format described in [VP9] Section 5.4 is used. Where only the framemarking RTP header extension is negotiated, non-hierarchical scalability modes SHOULD NOT be used (e.g. no support for the "S", KEY or KEY_SHIFT modes defined in [AV1] Section 6.7.5).

```

Comment:
Do you think it's necessary to talk about the frame marking extension in relation to the dependency descriptor? I feel like this is more of an academic scenario.

Reply:
framemarking-07 is supported in the webrtc.org code base and via the [RTP header extension API](https://w3c.github.io/webrtc-extensions/#rtcrtpheaderextensionparameters-interface) it's possible to Offer only framemarking or the Dependency Descriptor, or both. So it seems necessary to specify what happens when framemarking is Offered or is provided in an Answer.

Reply:
This text has been moved to PR https://github.com/AOMediaCodec/av1-rtp-spec/issues/97 so that it can be discussed separately.